tests/devices_test/lvm_test.py

```diff
@@ -87,13 +87,16 @@ def test_lvmcached_logical_volume_init(self):
         vg = LVMVolumeGroupDevice("testvg", parents=[pv, pv2])
 
         cache_req = LVMCacheRequest(Size("512 MiB"), [pv2], "writethrough")
-        lv = LVMLogicalVolumeDevice("testlv", parents=[vg],
+        lv = LVMLogicalVolumeDevice("testlv",
+                                    parents=[vg],
                                     fmt=blivet.formats.get_format("xfs"),
-                                    exists=False, cache_request=cache_req)
+                                    size=Size("2 MiB"),
```

Comment:
I think it would be better if the size here was expressed as `pv.format.min_size` or similar for clarity.

Reply:
Fixed.